dwave/system/testing.py

```diff
@@ -295,13 +322,17 @@ def __init__(self,
             'tags': [],
             'category': 'qpu',
             'quota_conversion_rate': 1,
+            'fast_anneal_time_range': [0.005, 83000.0],
```

Comment:
Should document in the pull request that this is a bugfix (missing property that triggers an error when running tests locally)